[2023-03-21 06:10:14,979] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: transform_etl.load_postgres manual__2023-03-21T06:09:20.019905+00:00 [queued]>
[2023-03-21 06:10:15,009] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: transform_etl.load_postgres manual__2023-03-21T06:09:20.019905+00:00 [queued]>
[2023-03-21 06:10:15,010] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 06:10:15,011] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2023-03-21 06:10:15,012] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 06:10:15,053] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): load_postgres> on 2023-03-21 06:09:20.019905+00:00
[2023-03-21 06:10:15,105] {standard_task_runner.py:52} INFO - Started process 138 to run task
[2023-03-21 06:10:15,111] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'transform_etl', 'load_postgres', 'manual__2023-03-21T06:09:20.019905+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpqorg6h6r', '--error-file', '/tmp/tmpbkspgfhg']
[2023-03-21 06:10:15,112] {standard_task_runner.py:77} INFO - Job 7: Subtask load_postgres
[2023-03-21 06:10:15,182] {logging_mixin.py:109} INFO - Running <TaskInstance: transform_etl.load_postgres manual__2023-03-21T06:09:20.019905+00:00 [running]> on host 1ee5a3730c13
[2023-03-21 06:10:15,263] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=transform_etl
AIRFLOW_CTX_TASK_ID=load_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T06:09:20.019905+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T06:09:20.019905+00:00
[2023-03-21 06:12:47,989] {python.py:152} INFO - Done. Returned value was: None
[2023-03-21 06:12:48,006] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=transform_etl, task_id=load_postgres, execution_date=20230321T060920, start_date=20230321T061014, end_date=20230321T061248
[2023-03-21 06:12:48,152] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-21 06:12:48,214] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
